error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -59,6 +60,33 @@ Mono<T> after(Supplier<? extends T> supplier) {
     }
   }
 
+  /** Prefer {@link Mono#empty()} over more contrived alternatives. */
+  static final class MonoEmpty<T> {
+    @BeforeTemplate
+    Mono<T> before() {
+      return Refaster.anyOf(Mono.just(null), Mono.justOrEmpty(null));
```

Comment:
I'm admittedly not fully sure whether we should _actually_ cover `Mono.just(null)`, so I added it as part of an extra commit. It's _such_ an obvious mistake that a Bug Check may make more sense.